shell/platform/windows/win32_window.cc

```diff
@@ -26,7 +18,11 @@ void Win32Window::InitializeChild(const char* title,
   Destroy();
   std::wstring converted_title = NarrowToWide(title);
 
-  WNDCLASS window_class = ResgisterWindowClass(converted_title);
+  // Set DPI awareness for all Windows versions. This call has to be made before
+  // the HWND is created.
+  dpi_helper_->SetDpiAwerenessAllVersions();
```

Comment:
As mentioned above, don’t set the DPI awareness mode here but in the calling app.